documentation/staging/content/managing-domains/domain-lifecycle/retry.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Domain failure retry processing"
+date: 2022-10-10T08:14:51-05:00
+draft: false
+weight: 8
+description: "This document describes domain failure retry processing in the Oracle WebLogic Server in Kubernetes environment."
+---
+
+{{< table_of_contents >}}
+
+The WebLogic Kubernetes Operator may encounter various failures during its processing of a Domain resource.
+Failures are are reported using Kubernetes events and the `spec.status` field in the Domain resource, 
+see [Domain debugging]({{< relref "/managing-domains/debugging#check-the-domain-status" >}}). 
+Failures fall into different categories and are handled differently by the operator, where most failures lead to automatic retries. 
+Refer to [Retry Behavior]({{< relref "#retry-behavior" >}}) on turning failure retry limits and intervals.
+
+### Domain failure severities
+
+Domain resource failures fall into the three severity levels:
+
+- Fatal failures
+   - Failures that any further retries will also fail unless the cause of the failure is fixed. \
+     Examples:
+     - Fatal errors in the introspector log that contain the special marker string `FatalIntrospectorError`.
+     - Validation errors in the domain resource that require changes to the domain spec.
+     - Failures at the `Severe` level that have reached the expected maximum retry time.
+- Severe failures
+   - Most of the failures during domain processing are temporary failures that may either resolve at a later 
```

Comment:
Consider adding sentence: The operator [periodically retries](#retry-behavior) when it encounters this type of failure.